config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -29,7 +29,7 @@
 
 # Do not treat an `ActionController::Parameters` instance
 # as equal to an equivalent `Hash` by default.
-# Rails.application.config.action_controller.allow_deprecated_parameters_hash_equality = false
+Rails.application.config.action_controller.allow_deprecated_parameters_hash_equality = false
```

Comment:
As far as I can tell we are not relying on this previous ability to compare params to hashes directly.